apps/emqx_rule_engine/src/emqx_rule_engine.erl

```diff
@@ -392,6 +392,7 @@ refresh_resource(#resource{id = ResId, config = Config, type = Type}) ->
     {ok, #resource_type{on_create = {M, F}}} = emqx_rule_registry:find_resource_type(Type),
         try cluster_call(init_resource, [M, F, ResId, Config])
         catch Error:Reason:ST ->
+            emqx_rule_monitor:ensure_resource_started(ResId),
             logger:critical(
                 "Can not re-stablish resource ~p: ~0p. The resource is disconnected."
```

Comment:
typo `re-stablish`, should be `reestablish`.

and maybe 'start' or 'create' is a better word ?

Reply:
I'll update this log message.